### PR TITLE
[10.0][MIG] partner_sale_risk

### PR DESCRIPTION
**Pull request by Ehtaga**
Split of the #442 PR. Depends on #451 .

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hi @Ehtaga ,

Are you going to finish this PR? If not please tell me

Thank you ! :smile:

**Review by carlosdauden — APPROVED** [review]

File: partner_sale_risk/views/res_partner_view.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <record id="res_partner_view_risk" model="ir.ui.view">
+        <field name="name">res.partner.view.risk</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="partner_financial_risk.res_partner_view_risk"/>
+        <field name="arch" type="xml">
+
+            <field name="risk_invoice_draft_include" position="before">
+                <field name="risk_sale_order_include"
+                       attrs="{'readonly': [('risk_allow_edit', '=', False)]}"/>
+                <field name="risk_sale_order" nolabel="1"/>
```
Comment (carlosdauden) [inline partner_sale_risk/views/res_partner_view.xml:15]: ¿Can you add new class implemented in #470?
class="oe_left_padding_15"

**Comment by pedrobaeza** [comment]
@chienandalu please superseed this PR rebasing over current 10.0 branch now that is has been merged and due to no answer by @Ehtaga

**Comment by Ehtaga** [comment]
@pedrobaeza  @rafaelbn @chienandalu Sorry I was not available those past days. I may be able to look at that later, but if @chienandalu can do it, it's perfect!

**Comment by chienandalu** [comment]
Ok, @Ehtaga it's on its way https://github.com/OCA/partner-contact/pull/589 :wink:

**Pull request closed without merging**